docs/sources/alerting/manage-notifications/manage-contact-points/_index.md

```diff
@@ -24,10 +25,10 @@ Complete the following steps to add a contact point.
 
 1. In the left-side menu, click **Alerts & incidents** and then **Alerting**.
 1. Click **Contact points**.
+1. From the **Choose Alertmanager** dropdown, select an Alertmanager. By default, **Grafana Alertmanager** is selected.
 1. Click **+ Add contact point**.
-1. From the **Alertmanager** dropdown, select an Alertmanager. By default, **Grafana Alertmanager** is selected.
 1. In **Name**, enter a descriptive name for the contact point.
-1. From **Contact point integration**, select a type and fill out mandatory fields. For example, if you choose email, enter the email addresses. Or if you choose Slack, enter the Slack channel(s) and users who should be contacted.
+1. From **Integration**, select a type and fill out mandatory fields. For example, if you choose email, enter the email addresses. Or if you choose Slack, enter the Slack channel(s) and users who should be contacted.
```

Comment:
The `<select/>` now just says "Integration".
![Screen Shot 2023-03-28 at 13 07 57](https://user-images.githubusercontent.com/41638679/228301721-e425dba8-385c-4ba8-a7e9-cef237f1938d.png)